includes/embeds/class-amp-twitter-embed.php

```diff
@@ -229,15 +229,31 @@ private function create_amp_twitter_and_replace_node( $dom, $node ) {
 			return;
 		}
 
+		$additional_attributes = [];
+
+		if ( $node->hasAttributes() ) {
+			foreach ( $node->attributes as $attr ) {
+				$name  = $attr->nodeName;
+				$value = $attr->nodeValue;
+
+				if ( substr( $name, 0, 5 ) === 'data-' ) {
```

Comment:
I wonder: should this not just go ahead and copy _all_ attributes? What if someone has a stylesheet that is targeting `.twitter-tweet`. Should not the `class` persist? What about other attributes?

Reply:
Good point, I now loop through all attributes.